### ISSUE TITLE
Does not overlay on top of Tabs

### ISSUE THREAD

**Issue by DavidTheProgrammer** [CarboniDavide/ion-bottom-sheet#3]
Hey, firstly, great work! Lovely plugin.

I've run into an issue where the bottom sheet doesn't overlay the bottom tabs. See attached image

![image](https://user-images.githubusercontent.com/18518167/107826286-53e0f480-6d8d-11eb-9fe3-a97abc3b1790.png)

**Comment by DavidTheProgrammer** [comment]
Found a workaround, this can be "fixed" by temporarily hiding the ion-tabs-bar while the ion sheet is visible.

**Comment by Saqib92** [comment]
Can you show me how did you manage to implement it. 
Showing error: 

`ion-bottom-sheet' is not a known element:
1. If 'ion-bottom-sheet' is an Angular component, then verify that it is part of this module.
2. If 'ion-bottom-sheet' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.

Can't bind to 'state' since it isn't a known property of 'ion-bottom-sheet'.`

**Comment by DavidTheProgrammer** [comment]
@Saqib92 Did you add `IonBottomSheetModule` to your imports in the module using the component?

**Comment by Saqib92** [comment]
Yes, I did import it in `app.module.ts`. Should i import it in `page.module` as well?

**Comment by DavidTheProgrammer** [comment]
@Saqib92 import it in the module declaring the component you want to use it in. So if you're using it in the `page.component`, import it in the `page.module`

**Comment by Saqib92** [comment]
Yes, it Worked. Thanks.

**Comment by Saqib92** [comment]
> Found a workaround, this can be "fixed" by temporarily hiding the ion-tabs-bar while the ion sheet is visible.

you can add it inside `tabs.html` it it will come over tabs.
